src/components/OutcomeCategorical/index.js

```diff
@@ -27,12 +27,23 @@ const OutcomeCategorical = ({ market, opts = {} }) => {
     return (
       <div className="outcomes outcomes--categorical">
         <div className="entry__color" style={{ backgroundColor: COLOR_SCHEME_DEFAULT[trendingOutcomeIndex] }} />
-        <div className="outcome">{ renderOutcomes[trendingOutcomeIndex] }</div>
+        <div className="outcome">{renderOutcomes[trendingOutcomeIndex]}</div>
         <div>
           {market.marginalPrices ? Math.round(market.marginalPrices[trendingOutcomeIndex] * 100).toFixed(0) : 0}%
         </div>
-        <div className="date">
-          { showDate ? moment(market.eventDescription.resolutionDate).format(dateFormat) : ''}
+        <div className="date">{showDate ? moment(market.eventDescription.resolutionDate).format(dateFormat) : ''}</div>
+      </div>
+    )
+  }
+
+  if (market.oracle.isOutcomeSet && market.oracle.outcome !== undefined) {
+    return (
+      <div className={`${className} outcomes outcomes--categorical`}>
+        <div className="outcome outcome__winning">
+          {renderOutcomes[market.oracle.outcome]}
+          <div className="outcome__bar--value">{`${Math.round(tokenDistribution[market.oracle.outcome] * 100).toFixed(
```

Comment:
Difficult to review, can you extract this calculation into a constant with a meaningful name?